tutorials/Measurements/Measurements.ipynb

```diff
@@ -0,0 +1,370 @@
+    {
+     "data": {
+      "application/json": "[\"Microsoft.Quantum.Standard::0.11.2003.3107\",\"Microsoft.Quantum.Katas::0.11.2004.2825\"]",
+      "text/html": [
+       "<ul><li>Microsoft.Quantum.Standard::0.11.2003.3107</li><li>Microsoft.Quantum.Katas::0.11.2004.2825</li></ul>"
+      ],
+      "text/plain": [
+       "Microsoft.Quantum.Standard::0.11.2003.3107, Microsoft.Quantum.Katas::0.11.2004.2825"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%package Microsoft.Quantum.Katas::0.11.2004.2825"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "C:/snippet_.qs(31,9): error QS6008: Multiple open namespaces contain a callable with the name \"H\". Use a fully qualified name instead. Open namespaces containing a callable with that name are Microsoft.Quantum.Intrinsic, Microsoft.Quantum.Primitive.\n",
+      "C:/snippet_.qs(35,13): error QS6008: Multiple open namespaces contain a callable with the name \"CNOT\". Use a fully qualified name instead. Open namespaces containing a callable with that name are Microsoft.Quantum.Intrinsic, Microsoft.Quantum.Primitive.\n",
+      "C:/snippet_.qs(40,33): error QS6008: Multiple open namespaces contain a callable with the name \"M\". Use a fully qualified name instead. Open namespaces containing a callable with that name are Microsoft.Quantum.Intrinsic, Microsoft.Quantum.Primitive.\n",
+      "C:/snippet_.qs(43,9): error QS6008: Multiple open namespaces contain a callable with the name \"ResetAll\". Use a fully qualified name instead. Open namespaces containing a callable with that name are Microsoft.Quantum.Intrinsic, Microsoft.Quantum.Primitive.\n",
+      "C:/snippet_.qs(11,9): error QS6008: Multiple open namespaces contain a callable with the name \"H\". Use a fully qualified name instead. Open namespaces containing a callable with that name are Microsoft.Quantum.Intrinsic, Microsoft.Quantum.Primitive.\n",
+      "C:/snippet_.qs(12,9): error QS6008: Multiple open namespaces contain a callable with the name \"CNOT\". Use a fully qualified name instead. Open namespaces containing a callable with that name are Microsoft.Quantum.Intrinsic, Microsoft.Quantum.Primitive.\n",
+      "C:/snippet_.qs(16,33): error QS6008: Multiple open namespaces contain a callable with the name \"M\". Use a fully qualified name instead. Open namespaces containing a callable with that name are Microsoft.Quantum.Intrinsic, Microsoft.Quantum.Primitive.\n",
+      "C:/snippet_.qs(18,9): error QS6008: Multiple open namespaces contain a callable with the name \"ResetAll\". Use a fully qualified name instead. Open namespaces containing a callable with that name are Microsoft.Quantum.Intrinsic, Microsoft.Quantum.Primitive.\n"
+     ]
+    }
+   ],
+   "source": [
+    "open Microsoft.Quantum.Canon;\n",
```

Comment:
You can get rid of those compilation errors if you drop those `open` directives - `Primitive` is deprecated in favor of `Intrinsic`, and that one and `Canon` are included in the notebooks by default

Reply:
Okay, got it will remove the unwanted dependencies.
Thanks again for helping me out